### PR TITLE
Update request and response to new contract, accept paging params

### PR DESCRIPTION
**Pull request by kelfink**

### CODE REVIEW

**Comment by kelfink** [comment]
retest this please

**Comment by kelfink** [comment]
retest this please

**Comment by kelfink** [comment]
retest this please

File: spec/acceptance/user_list_page_spec.rb [hunk 1]
```diff
@@ -66,15 +66,15 @@ def page_is_user_details
   end
 
   def page_count_users
-    page.all('.userRow').count
+    page.all('.rt-tr-group .link').count
```
Comment (eckdanny-osi) [inline spec/acceptance/user_list_page_spec.rb]: if we can, lets use non impl-specific targets so we're free to change w/o breaking acceptance tests.

**Pull request merged**